mercury_app/utils.py

```diff
@@ -304,14 +304,17 @@ def get_social_user(user_id):
     return social_user[0]
 
 
-def get_mock_api_event(amount):
+def get_mock_api_event(amount, event_id=0):
     ### UP TO 50 EVENTS ###
     mocked_events_array = []
     fake = Faker()
     for number in range(amount):
         created = fake.date_time_between(
             start_date='-5d', end_date='-1d', tzinfo=pytz.timezone('US/Eastern'))
-        eb_id = str(randint(50000000000, 60000000000))
+        if event_id == 0:
```

Comment:
have you tried using:
` range(1,amount)`
you can avoid this if

Reply:
What do you mean by replacing the `if` with `range(1,amount)`? The if is used to override the randomness in the `event id`. Thus, enabling for the creation of an event with a specific id for test purposes. There may be other cases where this is required, however up until now we have found just one use case.

Reply:
oooh my bad. Maybe you could just use a default random number instead of checking every time but it's up to you. Thank's for clarify this out

Reply:
We used to have a random number every time, but we found a case where an specific event id was required, hence the modification! Thanks for you input!